backend/metrics_backend/model/metrics_state.py

```diff
@@ -0,0 +1,68 @@
+import logging
+from typing import Dict
+
+from eth_utils import is_checksum_address
+from metrics_backend.utils import Address
+
+
+log = logging.getLogger(__name__)
+
+
+class MetricsState:
+    """ Manages the overall metrics. """
+
+    def __init__(self) -> None:
+        """ Initializes the metrics state. """
+
+        self.num_token_networks = 0
+        self.num_channels_opened = 0
+        self.num_channels_closed = 0
+        self.num_channels_settled = 0
+        self.open_channels_by_participant: Dict[Address, int] = dict()
+
+    def handle_channel_opened_event(
```

Comment:
Just an idea, but it might be helpful in the future if you also record the block numbers, That might make some nice stats.